docs/config.md

```diff
@@ -699,13 +740,13 @@ profile_models:
 More information about this can be found in the
 [Perf Analyzer documentation](https://github.com/triton-inference-server/client/blob/main/src/c++/perf_analyzer/README.md#ssltls-support).
 
-#### Important Notes:
+#### **Important Notes**:
 
-- Only a subset of flags can be specified on the command line. Use `model-analyzer profile --help` to see the list of flags that can be specified on the command line. If a flag isn't listed there, it can be specified via the yaml file.
+- Only a subset of flags can be specified on the command line. Use `model-analyzer profile --help` to see the list of flags that can be specified on the command line. If a flag isn't listed there, it can be specified via the YAML config file.
```

Comment:
Is this already stated higher in the file?

Reply:
It is. But I think it's important to restate this again for the user in this section, which is why I've left it in.